api/specs.proto

```diff
@@ -234,6 +246,14 @@ message NetworkSpec {
 	bool internal = 4;
 
 	IPAMOptions ipam = 5 [(gogoproto.customname) = "IPAM"];
+
+	// AllowContainers allows individual containers to be attached
+	// to this network. With this flag enabled individual
+	// containers running in an worker node in the cluster can
+	// attach to this network and access the services attached to
+	// this network. If this flag is not enabled(default case)
+	// independent containers cannot attach to this network.
+	bool allow_containers = 6;
```

Comment:
I don't have a silver bullet for the name, but here are a few ideas:

* global_network
* open_to_all_nodes
* worker_visible
* allow_worker_attach